web/themes/custom/pineapple/sass/layouts/nodes/_article.scss

```diff
@@ -13,24 +18,49 @@
     padding: $unit * 4;
     padding-top: 0;
     width: 100%;
-
-    @include breakpoint($desk) {
-      height: 400px;
-    }
+    height: auto;
 
     p {
       color: $primary;
     }
   }
 
   .teaser-image-container {
+    position: relative;
+    height: 100%;
+    flex-shrink: 0;
+    width: 100%;
+
+    @include breakpoint($palm) {
+    width: 33.33333%;
+    }
+
     img {
+      height: 100%;
       width: 100%;
       object-fit: cover;
+      display: none;
 
-      @include breakpoint($desk) {
-        width: auto;
+      @include breakpoint($palm) {
+        display: block;
       }
     }
   }
 }
+
+.article-teaser-container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  @include breakpoint($palm) {
+    flex-direction: row;
+  }
+
+  .views-row {
+    display: block;
+    margin-bottom: $unit * 3;
+    position: relative;
+    width: 100%;
+  }
+}
```

Comment:
This styling should not be in this file, it should be in a view related file as that's what it pertains to.

Reply:
done